app/views/decidim/action_delegator/admin/participants/index.html.erb

```diff
@@ -1,18 +1,38 @@
+<% missing_registered = missing_registered_users(current_setting&.participants) %>
+<% total_missing_emails = current_setting&.participants.where(email: "").count %>
 <div class="card with-overflow" id='user-groups'>
   <div class="card-divider">
-    <h2 class="card-title">
-      <%= link_to translated_attribute(current_setting.title), decidim_admin_action_delegator.settings_path %> &gt;
-      <%= t ".title" %>
-      <%= link_to t(".actions.new_participant"), decidim_admin_action_delegator.new_setting_participant_path(current_setting), class: "button tiny button--title" %>
-      <%= link_to t(".actions.csv_import"),
-                  decidim_admin_action_delegator.new_setting_manage_participant_path(setting_id: current_setting.id),
-                  class: "button tiny button--title" %>
-      <%= link_to t(".actions.remove_census"), decidim_admin_action_delegator.destroy_all_setting_manage_participants_path(current_setting),
-                  method: :delete,
-                  data: { confirm: t("participants.remove_census.confirm", scope: "decidim.action_delegator.admin") },
-                  class: "button tiny alert button--title" %>
+    <h2 class="card-title flex--sbc">
+      <div>
+        <%= link_to translated_attribute(current_setting.title), decidim_admin_action_delegator.settings_path %> &gt;
+        <%= t ".title" %>
+      </div>
+      <div>
+        <%= link_to t(".actions.new_participant"), decidim_admin_action_delegator.new_setting_participant_path(current_setting), class: "button tiny button--title" %>
+        <%= link_to t(".actions.csv_import"),
+                    decidim_admin_action_delegator.new_setting_manage_participant_path(setting_id: current_setting.id),
+                    class: "button tiny button--title" %>
+        <%= link_to t(".actions.remove_census"), decidim_admin_action_delegator.destroy_all_setting_manage_participants_path(current_setting),
+                    method: :delete,
+                    data: { confirm: t("participants.remove_census.confirm", scope: "decidim.action_delegator.admin") },
+                    class: "button tiny alert button--title" %>
+      </div>
     </h2>
   </div>
+  <% if missing_registered.present? %>
+    <div class="callout warning text-left">
+      <ul class="happy-faces">
+        <li>
+          <%= t(".missing_registered", missing_registered: pluralize(missing_registered.count, t(".participant"))) %>
+          <br>
+          <%= link_to t(".send_invitation_link"), invite_all_users_setting_invite_participants_path(current_setting), class: "action-icon--edit", method: :post %>
+        </li>
+        <% if Decidim::ActionDelegator.allow_to_invite_users && total_missing_emails.positive? %>
+          <li><%= t(".missing_emails", count: pluralize(total_missing_emails, t(".participant"))) %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
Let's put this in a particial like the `participants_sync_check`, also inside the "card-section" div.